src/TypeScript/Analyzer/JavaScriptAnalyzer.cs

```diff
@@ -67,7 +66,7 @@ internal sealed class JavaScriptAnalyzer : IAnalyzer, IDisposable
             IAnalysisConfigMonitor analysisConfigMonitor,
             ILogger logger)
             : this(eslintBridgeClientFactory, eslintBridgeProcess, activeRulesProvider,
-                new EslintBridgeIssueConverter(keyMapper.GetSonarRuleKey,
```

Comment:
Previously, this class was importing two different single-method interfaces, then passing both methods as delegates to the issue converter.

It was always the case that the two interfaces were implemented on the same underlying class, but it wasn't as obvious before. Now it's easier to see as we are passing _(ruleDefinitionsProvider.X, ruleDefinitionsProvider.Y)_.

We should just be able to pass the _ruleDefinitionsProvider_ instance to the issue converter, but we still can't do that. Still, getting rid of the key mapper interfaces simplifies things a bit.